datastore-api/app/core/wikidata.py

```diff
@@ -0,0 +1,211 @@
+import requests
+import json
+import concurrent.futures
+from typing import Tuple, Dict, List
+from collections import defaultdict
+from fastapi import Request, Response, HTTPException
+from fastapi.responses import JSONResponse
+from starlette.middleware.base import BaseHTTPMiddleware
+import aiohttp
+import asyncio
+
+import logging
+logger = logging.getLogger(__name__)
+
+class WikiData:
+    """Wraps access to the Bing Search API."""
+
+    kg_name = "wikidata-kg"    
```

Comment:
Better as `wikidata`, since the name of ConceptNet in SQuARE is `conceptnet` without "kg" in it